admin_view/nuxt-project/pages/assign_items/_id.vue

```diff
@@ -228,7 +228,7 @@
 
     <EditModal
       @close="closeItemEditModal"
-      v-if="isOpenItemEditModal"
+      v-if="isOpenItemEditModal && this.$role(this.roleID).rental_items.create"
```

Comment:
編集だからcreateよりupdateの方いいかも
```suggestion
      v-if="isOpenItemEditModal && this.$role(this.roleID).rental_items.update"
```